src/Mailer/Email.php

```diff
@@ -1925,6 +1929,25 @@ protected function _renderTemplates($content)
             $rendered[$type] = rtrim($rendered[$type], "\n");
         }
 
+        /* Embed images inline in html templates */
+        if (!empty($rendered['html'])) {
+            $rendered['html'] = str_replace(['file:', 'file://', 'cid://'], 'cid:', $rendered['html']);
```

Comment:
You missed the point of my comment. Currently this code will also replace the string in normal text for e.g. "Hey check this file: somename".  So instead of `str_replace()` you should use `preg_replace()` and ensure only strings in the `src` attribute of img tags are modified.